tests/Application/Observer/ProgressTest.php

```diff
@@ -155,7 +157,7 @@ private function getProperty($name)
 
     private function setOutputStream()
     {
-        $fp = fopen('data://text/plain;base64,', 'ab+');
+        $fp = fopen(self::TMP_FILE_PATH, 'w+');
```

Comment:
Indeed `data://` wrapper is limited to read-only starting from PHP 8.0
I suggest we use `php://memory` for this exact use case
```suggestion
        $fp = fopen('php://memory', 'ab+');
```
Without writing to the temporary filesystem.

Reply:
Ok, I will add this on a separate commit.